eng/mgmt/mgmtmetadata/compute_resource-manager.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=D:\hylee-sdk\may\sdk
-2020-05-26 00:22:02 UTC
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=D:\GitSwagger\azure-sdk-for-net\sdk
+2020-06-08 19:29:20 UTC
 Azure-rest-api-specs repository information
 GitHub fork: Azure
 Branch:      master
-Commit:      eb91865bf3284a739f1f0ded036f0e37bbcebe8d
+Commit:      8ba18e990a5c1a98d505018329d53272ce43f334
 AutoRest information
 Requested version: v2
-Bootstrapper version:    autorest@2.0.4413
+Bootstrapper version:    autorest@1.9.1
```

Comment:
Hi @chetmaddula I don't know why but your PR seemed to be generated with an old version of autorest. Could you run `autorest --reset` then run `generate.ps1` to generate again?
Not sure if the old version will cause any trouble but just in case. Thank you.

Reply:
will do and will update tags too